src/org/usfirst/frc/team4571/robot/Robot.java

```diff
@@ -10,7 +16,14 @@
  *
  */
 public class Robot extends IterativeRobot {
+	public static OperatorInterface oi;
```

Comment:
Can we delete the OperatorInterface class? I dont like it a lot. You can then delete this line from here